jobs/config.json

```diff
@@ -11173,6 +11173,22 @@
       "sig-node"
     ]
   },
+  "pull-kops-bazel-build": {
+    "_comment": "NOTE: args for this are in prow/config.yaml after the `--` !",
+    "args": [],
+    "scenario": "kubernetes_execute_bazel",
+    "sigOwners": [
+      "sig-cluster-lifecycle"
```

Comment:
sig aws

Reply:
the other kops jobs are sig-cluster-lifecycle. I think only kops-aws is aws...